src/bin/index.js

```diff
@@ -52,26 +52,31 @@ const run = async () => {
   if (options.courses && options.download) {
     await downloadCourses(options.courses, options.input)
   } else if (options.download && !options.courses) {
-    loggers.fileLogger.log({
-      level:   "error",
-      message: MISSING_JSON_ERROR_MESSAGE
-    })
     throw new Error(MISSING_JSON_ERROR_MESSAGE)
   }
-  writeBoilerplate(options.output).then(() => {
-    scanCourses(options.input, options.output, {
-      courses:      options.courses,
-      strips3:      options.strips3,
-      staticPrefix: options.staticPrefix
-    })
+  await writeBoilerplate(options.output)
+  await scanCourses(options.input, options.output, {
+    courses:      options.courses,
+    strips3:      options.strips3,
+    staticPrefix: options.staticPrefix
   })
 }
 
-run().catch(err => {
-  console.error("Error:", err)
-  loggers.fileLogger.log({
-    level:   "error",
-    message: err
+run()
+  .catch(err => {
+    console.error("Error:", err)
+    loggers.fileLogger.log({
+      level:   "error",
+      message: err.message
+    })
+    process.exit(1)
+  })
+  .then(() => {
+    if (loggers.memoryTransport.logs.length) {
+      console.error(
+        "Found errors which were logged: ",
```

Comment:
I'm not sure it's really necessary to log all these errors to console here.  There is a lot of problematic HTML we've yet to work through and the errors being swallowed up by turndown overflow my terminal buffer with even just the example courses.  With the full course load, this is going to dump a lot of text into the console log.  We should potentially log these to Sentry instead so they can be categorized in a sane manner.  There's no need to clog up the console with these, especially since you can't even scroll back and read all of them.

Reply:
I would like to have this information for debugging purposes. What do you think about leaving it out by default and having this output be enabled with a console argument, `--debug` or similar?

Reply:
That's a great idea, either `--debug` or `--verbose` would be fine.